### PR TITLE
Adresses

### PR DESCRIPTION
**Pull request by atodd-noms**
https://dsdmoj.atlassian.net/browse/NHPA-23

Display address list

### CODE REVIEW

**Comment by zali-moj** [comment]
looks fine
does it make sense to move the check for nulls/empty into the db abstraction layer?

File: test/subject.js [hunk 1]
```diff
@@ -49,10 +49,50 @@ describe('Subject data', function() {
...
 
             expect(err).to.be.null;
-
             expect(data).to.deep.equal(expectedInfo);
 
             done();
         });
     });
+
+    it.only("should return array of expected addresses objects", function(done) {
```
Comment (whitfield-mj) [inline test/subject.js]: Need to remove 'only' before merging

**Comment by atodd-noms** [comment]
Removed 'only' from the test
Moved null/empty checks to abstraction layer
Added display of the name associated with address where present

**Comment by zali-moj** [comment]
You could do this div= val.name instead of div #{val.name} since you're not manipulating string(s)

**Pull request merged**